usecases/nodes/handler.go

```diff
@@ -56,3 +57,15 @@ func (m *Manager) GetNodeStatus(ctx context.Context,
 	}
 	return m.db.GetNodeStatus(ctxWithTimeout, className, verbosity)
 }
+
+func (m *Manager) GetNodeStatistics(ctx context.Context,
+	principal *models.Principal,
+) ([]*models.Statistics, error) {
+	ctxWithTimeout, cancel := context.WithTimeout(ctx, GetNodeStatusTimeout)
+	defer cancel()
+
+	if err := m.authorizer.Authorize(principal, "list", "cluster"); err != nil {
```

Comment:
**question:**  shall it be ("list", "nodes") to be consistent with the upper method ?

Reply:
Yes it's there, but it looks like this:
```
"latest_configuration": "[{Suffrage:Voter ID:weaviate-1 Address:10.244.0.10:8300} {Suffrage:Voter ID:weaviate-0 Address:10.244.0.13:8300} {Suffrage:Nonvoter ID:weaviate-3 Address:10.244.0.12:8300} {Suffrage:Nonvoter ID:weaviate-2 Address:10.244.0.11:8300}]",
```
I have changed this so we would get a proper json array with objects in response like so:
```
"latestConfiguration": [
    {
        "address": "10.244.0.22:8300",
        "id": "weaviate-1",
        "suffrage": 0
    },
    {
        "address": "10.244.0.19:8300",
        "id": "weaviate-2",
        "suffrage": 0
    },
    {
        "address": "10.244.0.23:8300",
        "id": "weaviate-0",
        "suffrage": 0
    }
],
```

Reply:
I was thinking about this... I have made this as `list` `cluster` not `nodes` bc the statistics are being exposed by `/v1/cluster/statistics` endpoint not `/v1/nodes/statistics`